test/integration/src/py/pytest_utilities.py

```diff
@@ -32,6 +32,16 @@ def generate_minimal_test_account(
     return request, credentials
 
 
+def generate_sifchain_account(
+        base_transfer_request: EthereumToSifchainTransferRequest,
+        rowan_source_integrationtest_env_transfer_request: EthereumToSifchainTransferRequest,
+        rowan_source_integrationtest_env_credentials: SifchaincliCredentials,
+        target_ceth_balance: int = 10 ** 18,
+        target_rowan_balance: int = 10 ** 18
+):
+    assert False
+
```

Comment:
What is the purpose of this function?

Reply:
Cut and paste error, probably.  It's gone.